alchemist/alchemist-swingui/src/main/java/it/unibo/alchemist/boundary/interfaces/GraphicalOutputMonitor.java

```diff
@@ -25,51 +27,45 @@
     int getStep();
 
     /**
-     * @return true if this monitor is trying to draw in realtime
+     * @param step How many steps should be computed by the engine for the
+     *             display to update the graphics
      */
-    boolean isRealTime();
+    void setStep(int step);
```

Comment:
Is this used? Who calls it? Decision of whether to run or not a graphical update should be in charge of the UI and should not show in its interface

Reply:
Meh, it's in swingui. If it's unused in FX, leave it be.